### DISCUSSION TITLE
Is it possible to change Labs settings using Open GoPro API?

### DISCUSSION THREAD

**Discussion by aroooy** [gopro/labs#608] [Q&A] [answered]
I am developing a BLE remote control app for smartphones using Open GoPro API.  

I am asking this question after reading the documentation and knowing that this feature does not exist.  

Is it possible to check whether Labs is installed or change settings(BITR=150 etc.) etc. using the API?

Is there any chance this feature will be implemented?

**Comment by dnewman-gpsw** [comment, marked as answer]
I've also requested this from the Open GoPro team, as there isn't a way to do it. Please also request it at  https://github.com/gopro/OpenGoPro/discussions  ;)

**Reply by aroooy** [reply]
Thank you for your reply.  I'll also make a feature request :)

**Comment by tony-gutierrez** [comment]
I would also love this functionality! We are programmatically using cameras in a commercial application, and we also use gp labs features. We would love to be able to do all config via the api as opposed to also having to use qr codes.

**Reply by dnewman-gpsw** [reply]
Please add that request to the Open GoPro, I tease (partly), as this is existing request.  We need an Open GoPro message to Labs for this, hopeful it will happen.
